docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+# Example docker-compose file to run the Transition app in one container and a second one for PostGis
+#
+# Use the official postgis image for the DB, with the default username and password
+# The DB data live in a docker volume. You can adapt to have it live in a specific location
+# To run with an existing DB, change the PG_CONNECTION_STRING_PREFIX and drop the postgres service
+# and related volume
+#
+# It use the application configuration that was in the .env file that was built in the application image
+# It used "testtransition" as the application image, you can adapt to your prefered image name.
+
+version: "3.7"
+# docker-compose version 1.22 is required for this version
+
+
+services:
+  transition-www:
+    image: testtransition
+    ports:
+      - 8080:8080
+    environment:
+      PG_CONNECTION_STRING_PREFIX: postgres://postgres:pass@postgres:5432/
+    depends_on:
+      - postgres
+    #HACK, needed for running trRouting. See https://github.com/kaligrafy/trRouting/issues/23
+    tty: true
+    volumes:
+      # Fine-tune these paths to the target container's runtime directories
```

Comment:
We need to change those directory to something that make sense now. The example directory maybe?